sympy/physics/mechanics/tests/test_particle.py

```diff
@@ -33,6 +33,8 @@ def test_particle():
     assert p.angular_momentum(O, N) == m2 * r * (v3 * N.x - v1 * N.z)
     p.potential_energy = m * g * h
     assert p.potential_energy == m * g * h
+    assert p.total_energy(N) in [m*g*h + m2*(v1**2 + v2**2 + v3**2)/2,
+        m*g*h + m2 * v1**2 / 2 + m2 * v2**2 / 2 + m2 * v3**2 / 2]
```

Comment:
Why does this test say `assert f() in [f1, f2]`? Do  we not know which will be returned?

Reply:
Actually, this is the test I have taken from the current test under kinetic_energy() in ```mechanics/tests/test_particle.py```.  It has the following TODO: ```# TODO make the result not be system-dependent```. I thought it would be better to keep it until the current test is sorted. Could you suggest me how I can improve this.

Reply:
Presumably only one of those answers is returned so we don't need to test for one of two different answers.

Reply:
So, should I remove one of the tests according to which one I am getting on my machine?

Reply:
Yes. And if there are failures depending on Python-version one may add the second one back (right now we do not know that).

Reply:
@oscargus I corrected the test as desired. Please review.